package.json

```diff
@@ -89,7 +90,7 @@
     "semver": "7.3.8",
     "top-user-agents": "1.0.52"
   },
-  "resolutions": {
+  "overrides": {
```

Comment:
let's try to get rid of these if possible, at the very least the ESLint ones can be removed

Reply:
got this error message in a test
```
ERR_PNPM_INVALID_OVERRIDES_SELECTOR  Cannot parse the "tsdx/**/node-notifier" selector in pnpm.overrides
```
I thought I need at least the `tsdx/**/node-notifier` in `overrides`

Reply:
just remember this for later, once you add back the `overrides` that you need - we can probably get rid of the ESLint ones (also the `typescript-eslint` ones)